### PR TITLE
Suggested Story Draft 0.1

### PR DESCRIPTION
**Pull request by Khelthos**
+ Draft 0.1 for The Story

### CODE REVIEW

**Comment by davcri** [comment]
I freaking love this @Khelthos ! :smile:  
I'm okay to remove the old story and keep only this one in the GDD!

File: game-design-doc/game-design-document.md [hunk 1]
```diff
@@ -32,9 +32,18 @@
 Show how skilled you are and survive the waves of incoming spaceships! The more you survive, more points you get!
 
 # Story
+###### Suggested story
+War is changed. War is now a mere memory of what it was.
 
-The war is coming and everyone is trying to become a *space shooter*. 
+Year 322X. Major cities are now garbage dumps, the scum of the earth rules in every district and the high-ranking citizens live beyond the stratosphere where pollution and crime are miles away in suspended gardens on giants made of steel and concrete.
```
Comment (davcri) [inline game-design-doc/game-design-document.md]: Actually this description helps in setting the mood! :+1:

**Comment by Khelthos** [comment]
@davcri I didn't saw the conflict on commit, can you check it? Probably just merge or ignore my readme

**Comment by davcri** [comment]
@Khelthos I'll fix it, no problem!

**Comment by davcri** [comment]
Merged! Thanks for the great story and the help with the game design document! You are one of the few that contributed to it :)

For the name change I think it's a little bit late: one of the contributors made a custom logo for the project in #56

**Pull request merged**